File: Lexer/SemicolonToken.cpp
#include "token.h"
#include <string>
using namespace std;

class Semicolon : public Token {
	bool Token::equals()
	{

	}

	int Token::hashCode()
	{
		return 14;
	}

	string Token::toString()
	{
		return ";";
	}
};

File: src/main/java/parser/SemiColonToken.java
package parser;

public class SemiColonToken implements Token{
	public boolean equals(final Object other) {
		return other instanceof SemiColonToken;
	}
	
	public int hashCode() {
		return 11;
	}
	
	public String toString() {
		return "SemiColonToken";
	}

}
